BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -188,7 +188,12 @@ public final class BlueprintView: UIView {
     /// UILabel’s `intrinsicContentSize` behavior)
     public override var intrinsicContentSize: CGSize {
```

Comment:
Shouldn't we also be assigning the width and height that comes out of `return element.content.measure(`, eg if it returns `0` for a dimension? (Eg, imagine the element is an `Empty()`)

Reply:
I think in those cases `0` would be acceptable? The error we're trying to prevent only presents itself if there's not an element yet, right?

Reply:
Fair enough!